Library/KeyValueStoreType.swift

```diff
@@ -11,18 +11,18 @@ public enum AppKeys: String {
 }
 
 public protocol KeyValueStoreType: class {
```

Comment:
im tryna stay away from swift 3 naming guideline changes just to minimize the size of this diff. i wanna do the minimal work to get it compiling and working, and then we can do a separate swift 3 modernization pass later, which should be much easier